compiler/incremental-compilation-impl/test/org/jetbrains/kotlin/incremental/classpathDiff/ClasspathChangesComputerTest.kt

```diff
@@ -22,19 +21,18 @@ import kotlin.test.fail
 abstract class ClasspathChangesComputerTest : ClasspathSnapshotTestCommon() {
 
     // TODO Add more test cases:
-    //   - private/non-private fields
     //   - inline functions
```

Comment:
We compare Java classes and Kotlin classes separately so it will be treated as a removed Java class and an added Kotlin class (covered by `testModifiedAddedRemovedElements` for each language). But sure, I've added a TODO for this scenario.